packages/react-core/src/components/Page/PageHeader.tsx

```diff
@@ -22,11 +22,6 @@ export interface PageHeaderProps extends React.HTMLProps<HTMLDivElement> {
   showNavToggle?: boolean;
   /** True if the side nav is shown  */
   isNavOpen?: boolean;
-  /**
-   * If true, manages the sidebar open/close state and there is no need to pass the isNavOpen boolean into
-   * the sidebar component or add a callback onNavToggle function into the PageHeader component
-   */
-  isManagedSidebar?: boolean;
```

Comment:
In order to not break consumers who may have added this prop, can we add it back at mark it as deprecated with a console warning.

Reply:
sure! :+1: